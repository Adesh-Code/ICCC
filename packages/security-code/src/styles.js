const styles = ({ tokens, primitives }) => ({
  "&.DhigSecurityCode": {
    width: "56px",
    height: "72px",
    borderTop: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`,
    borderBottom: "none"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"][class*="MuiInputBase-inputAdornedEnd"]':
    {
      marginRight: "0"
    },
  "&.DhigSecurityCode[class*='MuiInput-root']:hover:before": {
    height: "2px",
    backgroundColor: tokens.colors["fill-engage"],
    boxShadow: "none",
    borderBottom: `${primitives.borderWidths.medium} solid ${primitives.colors.primary.black}`
  },
  "&.DhigSecurityCode:hover": {
    boxShadow: "none"
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]': {
    boxShadow: `0px 1px 0px 0px ${primitives.colors.primary.white}`
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]:before': {
    borderBottom: `1px solid ${primitives.colors.primary.black}`
  },
  "&.DhigSecurityCode input:focus": {
    boxShadow: `0px 2px 0px 0px ${tokens.colors["ink-focus"]} inset, 0px 0px 0px 2px ${tokens.colors["ink-focus"]} inset`
  },
  '&.DhigSecurityCode[class*="MuiInputBase-root"]:not(.Mui-disabled)::before': {
    boxShadow: "none"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"]:before': {
    width: "56px"
  },
  "&.DhigSecurityCode input": {
    flexGrow: "1"
  },
  '&.DhigSecurityCode [class*="MuiInputBase-input"]': {
    padding: "0",
    width: "56px",
    height: "72px",
    fontSize: `${primitives.fontSizeFallbacks["6"]}`,
    fontWeight: primitives.fontWeights.default
  },
  '&.DhigSecurityCode--dense [class*="MuiInputBase-input"]': {
    padding: "0",
    width: "32px",
    height: "40px",
    fontSize: `${primitives.fontSizeFallbacks["3"]}`,
    fontWeight: primitives.fontWeights.default
  },
  "&.DhigSecurityCode--dense": {
    width: "32px",
    height: "40px"
  },
  '&.DhigSecurityCode--error[class*="MuiInputBase-root"]:before': {
    borderBottom: `1px solid ${primitives.colors.accent.clay}`
  },
  "&.DhigSecurityCode--error, &.DhigSecurityCode:first-of-type.DhigSecurityCode--error, &.DhigSecurityCode:last-child.DhigSecurityCode--error, &.DhigSecurityCode--error:before":
    {
      borderColor: primitives.colors.accent.clay
    },
  '&.DhigSecurityCode--error[class*="MuiInputBase-root"]:after': {
    boxShadow: `0px 1px 0px 0px ${primitives.colors.accent.clay}`
  },
  "&.DhigSecurityCode:first-of-type": {
    borderLeft: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`
  },
  "&.DhigSecurityCode:last-child": {
    borderRight: `${primitives.borderWidths.medium} solid ${primitives.colors.tint["slate-450"]}`
  },
  "&.DhigSecurityCode:last-child [class*='MuiDivider-root'], &.DhigSecurityCode:focus-within [class*='MuiDivider-root']":
    {
      display: "none"
    }
});

export default styles;
