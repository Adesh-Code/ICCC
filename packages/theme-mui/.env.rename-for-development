# To load the MUI X Pro license for local development, rename this file to .env and replace with the actual license string
DHIG_MUIXPRO_ENTRY=PASTE_MUI_X_PRO_LICENSE_STRING_HERE