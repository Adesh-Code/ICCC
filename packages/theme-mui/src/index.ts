// Themes
export { default as themeDark } from "./themes/dark";
export { default as themeLight } from "./themes/light";

// Utilities
export { configureClassnameGenerator } from "./classname-generator";
export { setLicenseKey } from "./set-license-key";
