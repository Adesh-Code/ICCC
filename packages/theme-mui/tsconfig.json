{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "outDir": "lib/esm",
    "resolveJsonModule": true,
    "declarationDir": "lib/types"
  },
  "include": ["src"]
}
