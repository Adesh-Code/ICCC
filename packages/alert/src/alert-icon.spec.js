import React from "react";
import { render } from "@testing-library/react";
import { AlertIcon } from ".";

import { themeLight } from "@iccc/theme-mui";
import { createTheme, ThemeProvider } from "@mui/material/styles";

const light = createTheme(themeLight);

const arrange = ({ ...AlertIconProps }) =>
  render(
    <ThemeProvider theme={light}>
      <AlertIcon {...AlertIconProps} />
    </ThemeProvider>
  );

describe("Alert Icon component", () => {
  describe("when using a filled variant", () => {
    it("renders an error icon", () => {
      const { container } = arrange({ severity: "error", variant: "filled" });
      expect(container).toMatchSnapshot();
    });
    it("renders an info icon", () => {
      const { container } = arrange({ severity: "info", variant: "filled" });
      expect(container).toMatchSnapshot();
    });
    it("renders a success icon", () => {
      const { container } = arrange({ severity: "success", variant: "filled" });
      expect(container).toMatchSnapshot();
    });
    it("renders a warning icon", () => {
      const { container } = arrange({ severity: "warning", variant: "filled" });
      expect(container).toMatchSnapshot();
    });
    it("renders the default icon", () => {
      const { container } = arrange({ severity: null, variant: "filled" });
      expect(container).toMatchSnapshot();
    });
  });

  describe("when using a standard variant", () => {
    it("renders an error icon", () => {
      const { container } = arrange({ severity: "error", variant: "standard" });
      expect(container).toMatchSnapshot();
    });
    it("renders an info icon", () => {
      const { container } = arrange({ severity: "info", variant: "standard" });
      expect(container).toMatchSnapshot();
    });
    it("renders a success icon", () => {
      const { container } = arrange({
        severity: "success",
        variant: "standard"
      });
      expect(container).toMatchSnapshot();
    });
    it("renders a warning icon", () => {
      const { container } = arrange({
        severity: "warning",
        variant: "standard"
      });
      expect(container).toMatchSnapshot();
    });
    it("renders the default icon", () => {
      const { container } = arrange({ severity: null, variant: "standard" });
      expect(container).toMatchSnapshot();
    });
  });

  describe("when using a outlined variant", () => {
    it("renders an error icon", () => {
      const { container } = arrange({ severity: "error", variant: "outlined" });
      expect(container).toMatchSnapshot();
    });
    it("renders an info icon", () => {
      const { container } = arrange({ severity: "info", variant: "outlined" });
      expect(container).toMatchSnapshot();
    });
    it("renders a success icon", () => {
      const { container } = arrange({
        severity: "success",
        variant: "outlined"
      });
      expect(container).toMatchSnapshot();
    });
    it("renders a warning icon", () => {
      const { container } = arrange({
        severity: "warning",
        variant: "outlined"
      });
      expect(container).toMatchSnapshot();
    });
    it("renders the default icon", () => {
      const { container } = arrange({ severity: null, variant: "outlined" });
      expect(container).toMatchSnapshot();
    });
  });
});
