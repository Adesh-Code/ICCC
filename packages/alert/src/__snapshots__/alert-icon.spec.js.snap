// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert Icon component when using a filled variant renders a success icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="utility/complete"
      >
        <path
          d="M8.53113 12.5045L10.9428 14.775L15.7001 9.22485M22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12Z"
          id="stroke"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a filled variant renders a warning icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="utility/alert-outline"
      >
        <path
          d="M12 8.5V13.5M12 17V17.1111M12 1.5L23 20.2H1L12 1.5Z"
          id="stroke"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a filled variant renders an error icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="utility/error-outline"
      >
        <path
          d="M12 8V13M12 16.5V16.6111M12 2L22 12L12 22L2 12L12 2Z"
          id="stroke"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a filled variant renders an info icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="utility/info-outline"
      >
        <path
          d="M12 11V16M12 8V8.11111M22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12Z"
          id="stroke"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a filled variant renders the default icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="utility/info-outline"
      >
        <path
          d="M12 11V16M12 8V8.11111M22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12Z"
          id="stroke"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a outlined variant renders a success icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="complete"
      >
        <path
          clip-rule="evenodd"
          d="M17.625 33.75C26.5306 33.75 33.75 26.5306 33.75 17.625C33.75 8.71941 26.5306 1.5 17.625 1.5C8.71941 1.5 1.5 8.71941 1.5 17.625C1.5 26.5306 8.71941 33.75 17.625 33.75Z"
          fill="#2BC275"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M23.7823 12.4831C24.254 12.8875 24.3087 13.5977 23.9043 14.0694L16.7684 22.3947C16.5683 22.6281 16.2814 22.7695 15.9743 22.7859C15.6673 22.8024 15.3669 22.6924 15.1431 22.4817L11.5255 19.0759C11.0732 18.65 11.0517 17.938 11.4776 17.4856C11.9035 17.0332 12.6155 17.0118 13.0679 17.4377L15.8272 20.0355L22.196 12.6051C22.6003 12.1334 23.3106 12.0788 23.7823 12.4831Z"
          fill="black"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a outlined variant renders a warning icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="alert"
      >
        <path
          clip-rule="evenodd"
          d="M18.7757 2.07368L35.2021 29.9986C35.5551 30.5986 35.1225 31.3549 34.4264 31.3549H1.57355C0.877472 31.3549 0.444882 30.5986 0.797805 29.9986L17.2242 2.07368C17.5722 1.4821 18.4277 1.4821 18.7757 2.07368Z"
          fill="#FFC21A"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 11.63C18.6213 11.63 19.125 12.1337 19.125 12.755V20.255C19.125 20.8763 18.6213 21.38 18 21.38C17.3787 21.38 16.875 20.8763 16.875 20.255V12.755C16.875 12.1337 17.3787 11.63 18 11.63ZM18 24.38C18.6213 24.38 19.125 24.8836 19.125 25.505V25.6716C19.125 26.2929 18.6213 26.7966 18 26.7966C17.3787 26.7966 16.875 26.2929 16.875 25.6716V25.505C16.875 24.8836 17.3787 24.38 18 24.38Z"
          fill="black"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a outlined variant renders an error icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="inherit"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="error"
      >
        <path
          clip-rule="evenodd"
          d="M18.7425 2.24244L33.7575 17.2575C34.1676 17.6676 34.1676 18.3324 33.7575 18.7424L18.7425 33.7575C18.3324 34.1676 17.6676 34.1676 17.2575 33.7575L2.24246 18.7424C1.83241 18.3324 1.83241 17.6676 2.24246 17.2575L17.2575 2.24244C17.6676 1.83239 18.3324 1.83239 18.7425 2.24244Z"
          fill="inherit"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 10.875C18.6213 10.875 19.125 11.3787 19.125 12V19.5C19.125 20.1213 18.6213 20.625 18 20.625C17.3787 20.625 16.875 20.1213 16.875 19.5V12C16.875 11.3787 17.3787 10.875 18 10.875ZM18 23.625C18.6213 23.625 19.125 24.1286 19.125 24.75V24.9166C19.125 25.5379 18.6213 26.0416 18 26.0416C17.3787 26.0416 16.875 25.5379 16.875 24.9166V24.75C16.875 24.1286 17.3787 23.625 18 23.625Z"
          fill="currentColor"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a outlined variant renders an info icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="info"
      >
        <path
          clip-rule="evenodd"
          d="M18.005 34.13C26.9106 34.13 34.13 26.9106 34.13 18.005C34.13 9.09941 26.9106 1.88 18.005 1.88C9.09941 1.88 1.88 9.09941 1.88 18.005C1.88 26.9106 9.09941 34.13 18.005 34.13Z"
          fill="#5F60FF"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 25.6666C17.3787 25.6666 16.875 25.1629 16.875 24.5416L16.875 17.0416C16.875 16.4203 17.3787 15.9166 18 15.9166C18.6213 15.9166 19.125 16.4203 19.125 17.0416L19.125 24.5416C19.125 25.1629 18.6213 25.6666 18 25.6666ZM18 12.9167C17.3787 12.9167 16.875 12.413 16.875 11.7917L16.875 11.625C16.875 11.0037 17.3787 10.5 18 10.5C18.6213 10.5 19.125 11.0037 19.125 11.625L19.125 11.7917C19.125 12.413 18.6213 12.9167 18 12.9167Z"
          fill="white"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a outlined variant renders the default icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="info"
      >
        <path
          clip-rule="evenodd"
          d="M18.005 34.13C26.9106 34.13 34.13 26.9106 34.13 18.005C34.13 9.09941 26.9106 1.88 18.005 1.88C9.09941 1.88 1.88 9.09941 1.88 18.005C1.88 26.9106 9.09941 34.13 18.005 34.13Z"
          fill="#5F60FF"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 25.6666C17.3787 25.6666 16.875 25.1629 16.875 24.5416L16.875 17.0416C16.875 16.4203 17.3787 15.9166 18 15.9166C18.6213 15.9166 19.125 16.4203 19.125 17.0416L19.125 24.5416C19.125 25.1629 18.6213 25.6666 18 25.6666ZM18 12.9167C17.3787 12.9167 16.875 12.413 16.875 11.7917L16.875 11.625C16.875 11.0037 17.3787 10.5 18 10.5C18.6213 10.5 19.125 11.0037 19.125 11.625L19.125 11.7917C19.125 12.413 18.6213 12.9167 18 12.9167Z"
          fill="white"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a standard variant renders a success icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="complete"
      >
        <path
          clip-rule="evenodd"
          d="M17.625 33.75C26.5306 33.75 33.75 26.5306 33.75 17.625C33.75 8.71941 26.5306 1.5 17.625 1.5C8.71941 1.5 1.5 8.71941 1.5 17.625C1.5 26.5306 8.71941 33.75 17.625 33.75Z"
          fill="#2BC275"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M23.7823 12.4831C24.254 12.8875 24.3087 13.5977 23.9043 14.0694L16.7684 22.3947C16.5683 22.6281 16.2814 22.7695 15.9743 22.7859C15.6673 22.8024 15.3669 22.6924 15.1431 22.4817L11.5255 19.0759C11.0732 18.65 11.0517 17.938 11.4776 17.4856C11.9035 17.0332 12.6155 17.0118 13.0679 17.4377L15.8272 20.0355L22.196 12.6051C22.6003 12.1334 23.3106 12.0788 23.7823 12.4831Z"
          fill="black"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a standard variant renders a warning icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="alert"
      >
        <path
          clip-rule="evenodd"
          d="M18.7757 2.07368L35.2021 29.9986C35.5551 30.5986 35.1225 31.3549 34.4264 31.3549H1.57355C0.877472 31.3549 0.444882 30.5986 0.797805 29.9986L17.2242 2.07368C17.5722 1.4821 18.4277 1.4821 18.7757 2.07368Z"
          fill="#FFC21A"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 11.63C18.6213 11.63 19.125 12.1337 19.125 12.755V20.255C19.125 20.8763 18.6213 21.38 18 21.38C17.3787 21.38 16.875 20.8763 16.875 20.255V12.755C16.875 12.1337 17.3787 11.63 18 11.63ZM18 24.38C18.6213 24.38 19.125 24.8836 19.125 25.505V25.6716C19.125 26.2929 18.6213 26.7966 18 26.7966C17.3787 26.7966 16.875 26.2929 16.875 25.6716V25.505C16.875 24.8836 17.3787 24.38 18 24.38Z"
          fill="black"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a standard variant renders an error icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="inherit"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="error"
      >
        <path
          clip-rule="evenodd"
          d="M18.7425 2.24244L33.7575 17.2575C34.1676 17.6676 34.1676 18.3324 33.7575 18.7424L18.7425 33.7575C18.3324 34.1676 17.6676 34.1676 17.2575 33.7575L2.24246 18.7424C1.83241 18.3324 1.83241 17.6676 2.24246 17.2575L17.2575 2.24244C17.6676 1.83239 18.3324 1.83239 18.7425 2.24244Z"
          fill="inherit"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 10.875C18.6213 10.875 19.125 11.3787 19.125 12V19.5C19.125 20.1213 18.6213 20.625 18 20.625C17.3787 20.625 16.875 20.1213 16.875 19.5V12C16.875 11.3787 17.3787 10.875 18 10.875ZM18 23.625C18.6213 23.625 19.125 24.1286 19.125 24.75V24.9166C19.125 25.5379 18.6213 26.0416 18 26.0416C17.3787 26.0416 16.875 25.5379 16.875 24.9166V24.75C16.875 24.1286 17.3787 23.625 18 23.625Z"
          fill="currentColor"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a standard variant renders an info icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="info"
      >
        <path
          clip-rule="evenodd"
          d="M18.005 34.13C26.9106 34.13 34.13 26.9106 34.13 18.005C34.13 9.09941 26.9106 1.88 18.005 1.88C9.09941 1.88 1.88 9.09941 1.88 18.005C1.88 26.9106 9.09941 34.13 18.005 34.13Z"
          fill="#5F60FF"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 25.6666C17.3787 25.6666 16.875 25.1629 16.875 24.5416L16.875 17.0416C16.875 16.4203 17.3787 15.9166 18 15.9166C18.6213 15.9166 19.125 16.4203 19.125 17.0416L19.125 24.5416C19.125 25.1629 18.6213 25.6666 18 25.6666ZM18 12.9167C17.3787 12.9167 16.875 12.413 16.875 11.7917L16.875 11.625C16.875 11.0037 17.3787 10.5 18 10.5C18.6213 10.5 19.125 11.0037 19.125 11.625L19.125 11.7917C19.125 12.413 18.6213 12.9167 18 12.9167Z"
          fill="white"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;

exports[`Alert Icon component when using a standard variant renders the default icon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-0"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <svg
      fill="none"
      viewBox="0 0 36 36"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="info"
      >
        <path
          clip-rule="evenodd"
          d="M18.005 34.13C26.9106 34.13 34.13 26.9106 34.13 18.005C34.13 9.09941 26.9106 1.88 18.005 1.88C9.09941 1.88 1.88 9.09941 1.88 18.005C1.88 26.9106 9.09941 34.13 18.005 34.13Z"
          fill="#5F60FF"
          fill-rule="evenodd"
          id="fill-background"
        />
        <path
          clip-rule="evenodd"
          d="M18 25.6666C17.3787 25.6666 16.875 25.1629 16.875 24.5416L16.875 17.0416C16.875 16.4203 17.3787 15.9166 18 15.9166C18.6213 15.9166 19.125 16.4203 19.125 17.0416L19.125 24.5416C19.125 25.1629 18.6213 25.6666 18 25.6666ZM18 12.9167C17.3787 12.9167 16.875 12.413 16.875 11.7917L16.875 11.625C16.875 11.0037 17.3787 10.5 18 10.5C18.6213 10.5 19.125 11.0037 19.125 11.625L19.125 11.7917C19.125 12.413 18.6213 12.9167 18 12.9167Z"
          fill="white"
          fill-rule="evenodd"
          id="fill-foreground"
        />
      </g>
    </svg>
  </svg>
</div>
`;
